packages/calcite-components/package.json

```diff
@@ -25,8 +25,8 @@
     "build-storybook": "npm run util:build-docs && build-storybook --output-dir ./docs --quiet",
     "clean": "npm run util:clean-js-files && rimraf node_modules dist www hydrate docs .turbo src/components.d.ts",
     "deps:update": "updtr --exclude chalk cheerio typescript @types/jest jest jest-cli ts-jest puppeteer @whitespace/storybook-addon-html && npm audit fix",
-    "docs": "concurrently --kill-others --raw \"npm run build-storybook\" \"ts-node --esm ./support/cleanOnProcessExit.ts --path ./__docs-temp__\"",
-    "docs:preview": "concurrently --raw \"npm:util:build-docs && start-storybook\" \"ts-node --esm ./support/cleanOnProcessExit.ts --path ./__docs-temp__\"",
+    "docs": "concurrently --kill-others --raw \"npm run build-storybook\"",
+    "docs:preview": "concurrently --raw \"npm:util:build-docs && start-storybook\"",
```

Comment:
I don't think these need concurrently anymore now that we removed the cleanup script

Reply:
And can you append this to the scripts like in `start`:

```sh
npm run util:clean-js-files &&
```

Reply:
Nice catch. ✨🎣✨ Fixing!